### PR TITLE
Fix route integration test on ios

### PR DESCRIPTION
**Pull request by Jasguerrero**
## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: dev/devicelab/bin/tasks/route_test_ios.dart [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```
Comment (christopherfujino) [inline dev/devicelab/bin/tasks/route_test_ios.dart:1]: do you also have to update the .ci.yaml?

File: dev/devicelab/bin/tasks/route_test_ios.dart [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+import 'package:flutter_devicelab/framework/devices.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/task_result.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+import 'package:path/path.dart' as path;
+
+void main() {
+  task(() async {
+
```
Comment (christopherfujino) [inline dev/devicelab/bin/tasks/route_test_ios.dart:15]: nit: 
```suggestion
```

File: dev/devicelab/bin/tasks/route_test_ios.dart (hunk 1 above)
Comment (Jasguerrero) [reply dev/devicelab/bin/tasks/route_test_ios.dart:1]: Yes but only for the Mac-IOS; the Linux android stays the same

Comment (christopherfujino) [reply dev/devicelab/bin/tasks/route_test_ios.dart:1]: Oops, I didn't see that this PR had a diff on .ci.yaml. It was a long day yesterday :P

**Review by christopherfujino — APPROVED** [review]
LGTM

**Review by keyonghan — APPROVED** [review]
LGTM

**Pull request merged**